src/main/java/at/hannibal2/skyhanni/config/features/misc/PartyCommandsConfig.java

```diff
@@ -16,12 +16,17 @@ public class PartyCommandsConfig {
     @Expose
     @ConfigEditorBoolean
     @ConfigOption(name = "Party Transfer", desc = "Automatically transfer the party to people who type §b!ptme")
-    public boolean transferCommand = false;
+    public boolean transferCommand = true;
```

Comment:
Not sure about making this one `true` by default. This is an opt in in something that sends commands without user input (which can be construed to be against the rules)

Reply:
i guess thats fair, will change back